.github/workflows/ci-format.yml

```diff
@@ -36,4 +36,35 @@ jobs:
             -s n \
             --ignore marl_benchmark,examples,scenarios,docs,manager_pb2_grpc.py,worker_pb2_grpc.py \
             --msg-template='{path}: line {line}: {msg_id}: {msg}' \
-            ./smarts ./envision
\ No newline at end of file
+            ./smarts ./envision
+
+  test-types:
+    runs-on: ubuntu-18.04
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    container: huaweinoah/smarts:v0.4.18-minimal
+    steps:
+      - name: Install packages
+        run: |
+          add-apt-repository -y ppa:git-core/ppa
+          apt-get update
+          apt-get install -y git
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install SMARTS
```

Comment:
We need the dependendcies installed in the venv so that pytype can check the types. For example, if `gym` is not installed, `gym.Space` check will fail.